src/index.test.ts

```diff
@@ -1715,3 +1720,438 @@ describe('READY event emission', () => {
         expect(client.emit).toHaveBeenCalledWith(EVENTS.READY);
     });
 });
+
+describe('Experimental options togglesStorageTTL disabled', () => {
+    const lastUpdateStorageKey = 'repoLastUpdateTimestamp';
+
+    describe('Handling last update flag storage', () => {
+        let storageProvider: IStorageProvider;
+        let saveSpy: jest.SpyInstance;
+
+        class Store implements IStorageProvider {
+            public async save() {
+                return Promise.resolve();
+            }
+
+            public async get() {
+                return Promise.resolve([]);
+            }
+        }
+
+        beforeEach(() => {
+            storageProvider = new Store();
+            saveSpy = jest.spyOn(storageProvider, 'save');
+            jest.clearAllMocks();
+        });
+
+        test('Should not store last update flag when fetch is successful', async () => {
+            fetchMock.mockResponseOnce(JSON.stringify(data));
+
+            const config: IConfig = {
+                url: 'http://localhost/test',
+                clientKey: '12',
+                appName: 'web',
+                storageProvider,
+                experimental: {
+                },
+            };
+
+            const client = new UnleashClient(config);
+            await client.start();
+            expect(saveSpy).not.toHaveBeenCalledWith(
+                lastUpdateStorageKey,
+                {
+                    contextHash: expect.any(String),
+                    timestamp: expect.any(Number)
+                }
+            );
+        });
+    });
+  
+    test('Should not store last update flag when bootstrap is set', async () => {
```

Comment:
Est-ce que c'est vraiment ça qu'on teste ? En fait c'est parce qu'il n'y a pas l'option qu'on store pas non ?
```suggestion
    test('Should not store last update flag even when bootstrap is set', async () => {
```

Reply:
je vais modifier le nom , là on veut tester que l'on ne met pas a jour le TTL alors que l'on a un bootstrap de configuré (car on se trouve dans un describe avec la feature desactivé)